admin/test/test_packaging.py

```diff
@@ -542,19 +542,16 @@ def test_version_found(self):
         """
         ``GetPackageVersion`` assigns the exact version of a found package to
         its ``version`` attribute.
-
-        In particular, newer versions of pip/setuptools normalize the version
-        accoding to PEP440. We aren't prepared to handle that yet.
         """
         versions = [
             '0.3.2',
             '0.3.3dev5',
```

Comment:
There is a mix of "dev5" and ".dev5" being used. Is this intentional and if so, can you explain why?